### PR TITLE
Feature/cvsb 6660

### PR DESCRIPTION
**Pull request by KHodderDVSA**

### CODE REVIEW

File: tests/unit/generateNotificationData.unitTest.ts [hunk 1]
```diff
@@ -1,54 +1,58 @@
...
+                expect(sendNotificationData.startTimeDate).toEqual("14/01/2019");
+                expect(sendNotificationData.startTime).toEqual("08:47:33");
+                expect(sendNotificationData.endTime).toEqual("15:36:33");
+                expect(sendNotificationData.activityDetails.length).not.toEqual(0);
+                expect(countInstances(sendNotificationData.activityDetails,"---")).toEqual(2);
             });
         });
+
+        // TODO Add tests to verify the waitTime fields for "Time not Testing" activity added in ATF report.
+        // Verify for activityType, startTime, endTime, waitReason values.
```
Comment (agoncearenco) [inline tests/unit/generateNotificationData.unitTest.ts]: I see that the above tests include wait activities. Is this TODO still valid?

Comment (KHodderDVSA) [reply tests/unit/generateNotificationData.unitTest.ts]: replaced with test

File: package.json [hunk 2]
```diff
@@ -13,72 +13,64 @@
...
     "chai": "^4.2.0",
-    "gulp": "^4.0.0",
+    "gulp": "^4.0.2",
     "husky": "^3.0.9",
     "jest": "^24.8.0",
     "jest-plugin-context": "^2.9.0",
-    "nyc": "^14.1.1",
-    "serverless": "^1.42.3",
+    "serverless": "^1.56.1",
     "serverless-dependency-invoke": "^0.0.9",
```
Comment (agoncearenco) [inline package.json]: I'd suggest to also remove the dependency here

Comment (KHodderDVSA) [reply package.json]: It is - this is the "before" side.

Comment (agoncearenco) [reply package.json]: Sorry, I wasn't explicit enough. I saw that you removed the serverless-dependency-invoke config so I was suggesting to remove `"serverless-dependency-invoke": "^0.0.9"` also.

**Pull request merged**